src/components/Marketplace/ListingCard.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import Checkbox from '@material-ui/core/Checkbox';
+import PropTypes from 'prop-types';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Divider from '@material-ui/core/Divider';
+import AddListing from './AddListing';
+
+const useStyles = makeStyles({
+  listingCard: {
+    maxWidth: '70%',
+  },
+  listingCardPicture: {
+    height: '100%',
+  },
+  listingCardContent: {
+    backgroundColor: '#C4C4C4',
+    textAlign: 'left',
+    borderTop: '2px solid Gray',
+    borderBottom: '1px solid Gray',
+    paddingTop: '8px',
+  },
+  listingCardActionArea: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    height: '100%',
+    width: '100%',
+    padding: '0px',
+    backgroundColor: '#C4C4C4',
+  },
+  divider: {
+    backgroundColor: 'Gray',
+  },
+});
+export default function ListingCard({
+  id, getListing, onSelect, selected, // record,crop, unitsInStock, price, unitsPending,
+}) {
+  const classes = useStyles();
+  const [editActive, setEditActive] = useState(false);
+  const listing = getListing(id);
+  // const [editRecord, setEditRecord] = useState(listing);
+  const handleClickOpen = () => {
+    setEditActive(true);
+  };
+
+  const handleClose = () => {
+    setEditActive(false);
+  };
+  return (
+    <>
+      <Card className={classes.listingCard}>
+        <CardActionArea>
+          <CardMedia
+            className={classes.listingCardPicture}
+            image="/bananas-sample.png"
```

Comment:
For images, let's make a directory in src called assets, and inside that create a directory called images. We can store the banana pic in there for now and import it in this file